docs/configuration/backends/kubernetes.md

```diff
@@ -50,6 +50,16 @@ See also [Kubernetes user guide](/user-guide/kubernetes).
 #
 # labelselector = "A and not B"
 
+# Value of `kubernetes.io/ingress.class` annotation that identifies Ingress objects to be processed.
+# Note that Ingress objects with an empty or nonexistent `kubernetes.io/ingress.class` value
+# will also be processed.
```

Comment:
The section might need to be updated if my assumption on how we should behave (see my other comment below) is correct.